src/app/types.ts

```diff
@@ -1,5 +1,27 @@
+import type { WordList } from "@common/types";
 export interface DictionarySource {
   name: string;
   size: number;
   type: string;
 }
+export interface StoredWordList {
+  /** Autogenerated primary key */
+  id?: number;
+
+  /**
+   * Typically the filename of the uploaded dictionary source.
+   */
+  name: string;
+  /**
+   * The actual contents of said file.
+   */
+  wordlist: WordList;
+  /**
+   * Size of the file in bytes.
```

Comment:
**words** is a much more useful thing to store than bytes, for the end-user perspective!

```suggestion
   * Size of the file in words.
```